src/test/kv_store_bench.cc

```diff
@@ -115,7 +115,9 @@ int KvStoreBench::setup(int argc, const char** argv) {
       } else if (strcmp(args[i], "--cache-size") == 0) {
 	cache_size = atoi(args[i+1]);
       } else if (strcmp(args[i], "--cache-refresh") == 0) {
-	cache_refresh = 100 / atoi(args[i+1]);
+	auto temp = atoi(args[i+1]);
+	assert (temp != 0);
```

Comment:
remove the space after `assert`. please return failure `temp` is 0.

Reply:
@tchaikov Thanks for review.
1. Will remove space after `assert`
2. > please return failure if temp is 0.
assert(expression). If expression evaluates to false, program is terminated.
With `assert(temp != 0)` Here if temp becomes 0 code will assert. I believe this is correct condition.

What code piece you suggest?

Reply:
@amitkumar50 
```c++
if (!temp) {
  return 1;
}
```